src/SFA.DAS.EmployerIncentives.Web/ViewModels/Home/HomeViewModel.cs

```diff
@@ -1,20 +1,57 @@
-﻿namespace SFA.DAS.EmployerIncentives.Web.ViewModels.Home
+﻿using System;
+
+namespace SFA.DAS.EmployerIncentives.Web.ViewModels.Home
 {
     public class HomeViewModel : IViewModel
     {
-        public string AccountId { get; }
-        public string AccountLegalEntityId { get; }
-
         public string Title => "Apply for the hire a new apprentice payment";
 
+        public string AccountId { get; }
+        public string AccountLegalEntityId { get; }
+        public bool NewAgreementRequired { get; }
+        public bool BankDetailsRequired { get; }        
+        public string AccountsAgreementsUrl { get; }
         public string OrganisationName { get; set; }
 
-        public HomeViewModel(string accountId, string accountLegalEntityId, string organisationName)
+        private string ManageApprenticeshipSiteUrl { get; }
+        private const string RejectedDataValidation = "Case Rejected - Data validation";
+        private const string RejectedVer1 = "VER1 Rejected";
+        private const string RejectedVerification = "Case Rejected - Verification";
+
+        public HomeViewModel(
+            string accountId, 
+            string accountLegalEntityId, 
+            string organisationName,             
+            bool newAgreementRequired, 
+            string manageApprenticeshipSiteUrl,
+            string vrfCaseStatus,
+            string vrfVendorId)
         {
             AccountId = accountId;
             AccountLegalEntityId = accountLegalEntityId;
             OrganisationName = organisationName;
+            NewAgreementRequired = newAgreementRequired;
+            ManageApprenticeshipSiteUrl = manageApprenticeshipSiteUrl;
+            if (!manageApprenticeshipSiteUrl.EndsWith("/"))
+            {
+                ManageApprenticeshipSiteUrl += "/";
+            }
+            AccountsAgreementsUrl = $"{manageApprenticeshipSiteUrl}accounts/{AccountId}/teams";
+            BankDetailsRequired = MapBankDetailsRequired(vrfCaseStatus, vrfVendorId);
         }
 
+        // TODO: move to api
```

Comment:
Could bank details required be a property of the LegalEntity?

Reply:
Yes it should/could be.  Is a refactor task though.

Reply:
Have we got a ticket for it?

Reply:
I've moved it to the model to encapsulate it better.  Will raise a ticket to get the api to calculate and return it.

Reply:
https://skillsfundingagency.atlassian.net/browse/EI-1179